src/screens/Login/index.tsx

```diff
@@ -25,7 +26,7 @@ class Login extends Component {
   handleChange = event => {
     this.setState({
       [event.target.id]: event.target.value
-    });
+    } as Pick<ILoginStates, keyof ILoginStates>);
```

Comment:
What's this used for?

Reply:
I try to tell that this property belong to ILoginStates